plasma_framework/migrations/2_deploy_plasma_framework.js

```diff
@@ -19,4 +21,6 @@ module.exports = async (
 
     const plasmaFramework = await PlasmaFramework.deployed();
     await plasmaFramework.activateChildChain({ from: authorityAddress });
+    const sha = childProcess.execSync('git rev-parse HEAD').toString().trim().substring(0, 7);
+    await plasmaFramework.setVersion(''.concat(pck.version, '+', sha), { from: maintainerAddress });
```

Comment:
Nitpick: javascript convention that we follow (more or less) is to use string interpolation instead of `.concat`when building strings
https://github.com/airbnb/javascript#es6-template-literals

```suggestion
    await plasmaFramework.setVersion(`${pck.version}+${sha}`), { from: maintainerAddress });
```